pytubefix/contrib/search.py

```diff
@@ -14,7 +14,8 @@
 class Search:
     def __init__(
             self, query: str,
```

Comment:
I think it would be more intuitive for the user to leave the client as the second parameter.